### PR TITLE
new role variables to allow setting custom scala+sbt install folders

### PR DESCRIPTION
**Pull request by lhoss**

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -11,5 +11,5 @@ galaxy_info:
     - trusty
   categories:
   - development
-dependencies:
-  - { role: "azavea.java" }
+#dependencies:
```
Comment (hectcastro) [inline meta/main.yml]: Are you installing `java6-runtime-headles` or equivalent on your own? The role fails to provision without it.

Comment (lhoss) [inline meta/main.yml]: Indeed, we don't want to hardcode such a base dependency (like java) that is used by almost any framework/tool.

ps: I'm still waiting for any fix to: 

> Unfort. ansible doesn't yet offer a feature to easily disable 'transient dependencies'.
> I created an ansible feature request: ansible/ansible#13215 , but will probably take time to get it builtin (hopefully for v2.1)

Comment (lhoss) [reply meta/main.yml]: @hectcastro would you merge the PR, if undo this disabled java dependency?
ps:  Unfort. my issue (https://github.com/ansible/ansible/issues/13215) didn't get enough traction .. at least there's more recent similar requests that would also help, like https://github.com/ansible/ansible/issues/34566

Comment (lhoss) [reply meta/main.yml]: I've now done as I proposed -^ in a new PR, AND also properly documenting the 3 new role vars
-> https://github.com/azavea/ansible-scala/pull/4 👍

**Comment by lhoss** [comment]
replaced by https://github.com/azavea/ansible-scala/pull/4

**Pull request closed without merging**